Source/Scene/Camera.js

```diff
@@ -3181,6 +3181,23 @@ Camera.prototype.cancelFlight = function () {
   }
 };
 
+/**
+ * Complete the current camera flight if one is in progress
+ * The camera immediately move to destination
+ */
+Camera.prototype.completeFlight = function () {
+  if (defined(this._currentFlight)) {
+    this.cancelFlight();
+
+    newOptions.duration = 0;
+    this.setView(newOptions);
+
+    if (defined(newOptions.complete)) {
```

Comment:
To be safe, I would use `this._currentFlight.complete()` instead.  Then also set `this._currentFlight` to undefined, since the flight is done.